.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## How do I contribute?
+
+There are two logical avenues to contribute to the code and content usrse.github.io, including (but not limited to):
```

Comment:
I would change "avenues" to "methods" to be slightly more clear

Reply:
How about "ways" ?

Reply:
sure

Reply:
I changed this to "ways" and made all the other edits as suggested - let me know if it doesn't sound good with "ways."